### ISSUE TITLE
Dev-Match 머신러닝 프로젝트 3차 시도 : 검증 정확도 높이기 성공

### ISSUE THREAD

**Issue by RyanKor** [RyanKor/dev-match-ML#4]
# 4번째 CNN 학습 진행 결과 (validation data set 분리 & 미니 배치 크기 변경, 배치 정규화 도입 등)

- 기존 training data에서 20% 만큼을 validation 데이터 셋으로 분류 진행 (데이터 자체가 워낙 적다보니, 기존에 코세라에서 배운 것처럼 10%를 쪼개서 validation set으로 사용할 경우, 충분히 validate가 안될 것이라 판단)

- 이후, 기존에 Image Generator에 적용되어 있던 미니 배치 크기 변경 (1 -> 64 -> 128로 단계적으로)

- 배치 정규화 적용 (각 신경망에서 출력된 결과 값의 정규화 적용)

- 활성화 비율이 낮은 노드에 대해 dropout 사용

등으로 결과를 조율했더니 학습 정확도 향상에 따른 validation 정확도가 함께 향상되었다.

그러나 아직 Test 결과 값과의 정확도 차이가 많이 나서 이제 이 부분을 조율해보는 단계를 거쳐보자.

이루지 못 할 일은 없다.

![image](https://user-images.githubusercontent.com/40455392/133996171-fa65334c-7b49-4581-83df-e92d84c477ad.png)
